### PR TITLE
feat: html css tutorials

### PR DESCRIPTION
**Pull request by Macdeal**
please review this

### CODE REVIEW

File: tut4-css/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<html>
+
+    <head>
+        <link rel="stylesheet" href="style.css">
+
```
Comment (donjude-eng) [inline tut4-css/index.html:7]: can remove empty space?

File: tut5-css/selector.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <html>
+
+        <head>
+            <link rel="stylesheet" href="style.css">
+            
+            <title>  learning CSS </title>
+            
+        </head>
+    <body>
+        <h1><strong>THE GREAT INDIAN FESTIVAL </strong></h1>
```
Comment (donjude-eng) [inline tut5-css/selector.html:12]: always good to split to

```
<header> - Defines a header for a document or a section
<nav> - Defines a set of navigation links
<section> - Defines a section in a document
<footer> - Defines a footer for a document or a section
```

File: tut6-10/index.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          </form>
+
+      </section>
+      <footer>
+          <p class="copyrite">Copyrite 2018 Macdeal</p>
+      </footer>
+          
+
+  </body>
+
```
Comment (donjude-eng) [inline tut6-10/index.html:75]: remove this line

**Review by donjude-eng — CHANGES_REQUESTED** [review]

File: tut5-css/selector.html (hunk 2 above)
Comment (Macdeal) [reply tut5-css/selector.html]: ok

File: tut6-10/index.html (hunk 3 above)
Comment (Macdeal) [reply tut6-10/index.html]: yes

File: tut4-css/index.html (hunk 1 above)
Comment (Macdeal) [reply tut4-css/index.html]: yes i will do

**Review by donjude-eng — APPROVED** [review]

**Pull request merged**